src/sql/workbench/services/connection/browser/media/connectionDialog.css

```diff
@@ -137,6 +137,10 @@
 	display: none;
 }
 
+.hide-trustServerCertificate .option-trustServerCertificate {
```

Comment:
These should be generated dynamically at runtime using [createCSSRule](https://github.com/Microsoft/azuredatastudio/blob/main/src/vs/base/browser/dom.ts#L805)